bugbug/tools/code_review.py

```diff
@@ -1401,6 +1407,9 @@ def _get_comment_examples(self, patch, created_before: datetime | None = None):
 
         if not comment_examples:
             comment_examples = STATIC_COMMENT_EXAMPLES
+        else:
+            for example in comment_examples:
+                example["comment"]["explanation"] = "THE JUSTIFICATION GOES HERE"
```

Comment:
Here we should reuse the old generated explanations when there are some

Reply:
I'm not sure that would add value, especially since it will not be consistent when some comments have justification value and others do not. But it will definitely consume more tokens.

I added the placeholder here to avoid situations where the explanation is omitted.

Reply:
Let's file a follow-up issue to consider adding it (and we could auto-generate it when it is missing)

Reply:
I filed https://github.com/mozilla/bugbug/issues/5017